src/CircleEditor/CircleEditorDocument.ts

```diff
@@ -367,4 +368,331 @@ export class CircleEditorDocument extends Disposable implements vscode.CustomDoc
     let responseJson = {command: 'loadJson', data: jsonModel};
     this._onDidChangeContent.fire(responseJson);
   }
+
+  /**
+   * Guess data's type (int or float)
+   */
+  private guessExactType(n: any) {
+    if (isNaN(Number(n))) {
+      return 'error';
+    }
+
+    if (Number(n) % 1 === 0) {
+      return 'int';
+    } else if (Number(n) % 1 !== 0) {
+      return 'float';
+    }
```

Comment:
For consistent return type :D

```suggestion
    } else {
      return 'not supported type';
    }
```

Reply:
I modified that logic.